README.md

```diff
@@ -18,3 +18,91 @@ is created
 ### To run the tests locally
 
 1. Run `npm test`
+
+## Before building a docker image
+
+Before deployment of a new Docker container to the kubernetes cluster, check that the following are in place
+
+### Frontend artifacts
+
+The ODetect front end is built using Angular JS. Once built locally, the front end artifacts are stored in the `/Public/ODetect` folder.
+
+In case you are not building the front end locally, you may download a zip file of the front end artifacts from the shared ODetect vault on 1password and unzip to the above folder.
```

Comment:
Does this mean that you're going to upload the updated zip every time you change something on the front end?
Is that step documented anywhere?
It sounds very fragile. If building the front-end artifacts isn't difficult, it might be better just to have everyone do it.

Reply:
I agree with this - it's a single command run from inside the frontend source code file. Then copying this into the Public/ODetect folder.

https://github.com/bravetechnologycoop/ODetect-Frontend-Local

so npx @angular/cli build --prod && mv dist/ODetect ../ODetect-Backend-Local/Public/ODetect

could do the trick

Reply:
@mariocimet i've updated the readme, could you check if the build commands look ok?

Reply:
Now that the instructions for building the front end are here, I'd suggest removing the instructions related to 1password and delete the zip from 1password itself. Otherwise, it could get out of date and confusing.

Reply:
Done.